### PR TITLE
New: Allow visua11y to appear in drawer

### LINKED ISSUE #85: Allow visua11y to appear in drawer or notify
### Subject of the issue
Allow visua11y to appear in drawer or notify

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #85 

### New
* Allow visua11y to appear in drawer

### Includes
* https://github.com/cgkineo/adapt-visua11y/pull/88 Breakpoint fixes

### CODE REVIEW

**Comment by oliverfoster** [comment]
@kirsty-hames would you mind styling this properly please?

**Comment by joe-replin** [comment]
Functionally, this works well and I like the solution for solving #83. With screen readers, I've noted a few issues.

Testing with NVDA with either Chrome or FF
- The entirely of the contents of the visua11y drawer is read out after the drawer heading is announced. 
- Tabbing forward passed the close button focuses on the grouping for the entire visua11y drawer. The entirely of its contents is read out a 2nd time.

Testing with JAWS with either Chrome or FF
- The current browser tab is being read out in addition to the drawer header. 
- On first tab forward in focusing on the close button, the entirely of the visua11y drawer is being read out. 
- Attempting to use arrow keys to toggle radio buttons in the readability group first causes the label to be spelled out and then causes further keyboard presses to become unresponsive.

**Comment by joe-replin** [comment]
Sure thing, @oliverfoster. Reviewing the PR was just the last thing I did at EOD yesterday so wasn't solutioning just yet. :)

**Comment by oliverfoster** [comment]
Actually, on second read, that stuff does sound quite fundamental. I'll do it, don't worry.

**Comment by kirsty-hames** [comment]
For `_location` `notify`, we have a top level class `is-visua11ysettings` applied to `.notify`. This is useful if you want style the Visua11y popup different from other notify popups in Adapt. Should we apply the same for `.drawer`?

File: less/visua11ySettings.less [hunk 1]
```diff
@@ -1,21 +1,114 @@
...
     padding: @item-padding (@item-padding / 2);
     border-radius: 3px;
   }
 }
 
-// UI settings
+// Notify view
 // --------------------------------------------------
-.visua11ysettings {
+.notify__popup.is-visua11ysettings {
```
Comment (kirsty-hames) [inline less/visua11ySettings.less:17]: This selector `.notify__popup.is-visua11ysettings`, should be replaced with `.visua11ysettings-notify` once added (for consistency with drawer selector `.visua11ysettings-drawer`).

Comment (joe-replin) [reply less/visua11ySettings.less]: I've committed this change.

**Review by kirsty-hames — CHANGES_REQUESTED** [review]
Working as expected thanks. Testing on Mac Safari, FF, Chrome and iPhone.

File: less/visua11ySettings.less [hunk 2]
```diff
@@ -1,21 +1,114 @@
...
     padding: @item-padding (@item-padding / 2);
     border-radius: 3px;
   }
 }
 
-// UI settings
+// Notify view
 // --------------------------------------------------
-.visua11ysettings {
+.notify__popup.visua11ysettings-notify {
```
[the remark this answers is not in the document]
Comment (joe-replin) [reply less/visua11ySettings.less]: Great call. Committed a change.

**Review by swashbuck — APPROVED** [review]
👍

**Pull request merged**